src/main/js/controls/Graph/Graph.js

```diff
@@ -327,17 +329,21 @@ class Graph extends Construct {
      * The content serves as a 1to1 relationship. For rendering
      * multiple data sets respective number of content needs to be provided.
      *
-     * @param {object} content - Graph content to be removed
+     * Input can be either a GraphContent instance or
+     * just an object containing a `key` of the content to be removed
+     *
+     * @param {object} input - Graph content to be removed
      * @returns {Graph} - Graph instance
      */
-    unloadContent(content) {
-        const index = this.content.indexOf(content);
+    unloadContent(input) {
```

Comment:
Is it only for graphs ? Didn't Gantt have a similar requirement ??

Reply:
For Gantt, Timeline and Pie it is enough to provide key and label for unloading. 
For Graph however, we are currently requiring consumer to provide the entire object. 

Consider this, 
```
var graph = new Graph();
var line1 = new Line(input);
graph.loadContent(line1);
```
For unloading you need to provide the same exact object. 
```
graph.unloadContent(line1);
```
This means that if I were to do `newLine1` with exact same input. 
```
var newLine1 = new Line(input);
graph.unloadContent(newLine1);
```
it wouldn't work , since Objects in JavaScript are compared by reference and not value.